include/api/wasmedge/wasmedge.h

```diff
@@ -1166,6 +1166,14 @@ WasmEdge_GlobalTypeCreate(const enum WasmEdge_ValType ValType,
 WASMEDGE_CAPI_EXPORT extern enum WasmEdge_ValType
 WasmEdge_GlobalTypeGetValType(const WasmEdge_GlobalTypeContext *Cxt);
 
+/// Get the full extended value type from a global type.
+///
+/// \param Cxt the WasmEdge_GlobalTypeContext.
+///
+/// \returns the full extended value type of the global type.
+WASMEDGE_CAPI_EXPORT WasmEdge_FullValType
+WasmEdge_GlobalTypeGetFullValType(const WasmEdge_GlobalTypeContext *Cxt);
```

Comment:
You are introducing a new public API; however, there is no detailed explanation for the difference between the `ValType` and `FullValType.`

Reply:
This function is not compulsory for this PR. Do you think we should remove this function for this PR and implemented a list of new public API for this refactor in a separate PR. @hydai

Reply:
It would be better if we collect all changes of the public API in another PR. So we can make sure the document and APIs are all synced and updated.

Reply:
I see. I will remove this function.